app/helpers/application_helper.rb

```diff
@@ -765,7 +765,7 @@ def display_back_button?
   end
 
   def display_adv_search?
-    %w(availability_zone cloud_volume container_group container_node container_service
```

Comment:
You are starting new controller, maybe we your use this opportunity to cleanup this a bit and introduce something like `controller.had_adv_search?` and test for that first. Instead of adding to the ugly list we have here.
Does it make sense? Guess there will not be advanced search for any of this controller's screens, right?